testcases/basic/rcfile

```diff
@@ -0,0 +1,27 @@
+# Clear any old environment that may conflict.
+for key in $( set | awk '{FS="="}  /^OS_/ {print $1}' ); do unset $key ; done
+export OS_USERNAME=Dev
+export OS_USER_DOMAIN_NAME=Default
+export OS_PROJECT_DOMAIN_NAME=Default
+export OS_BAREMETAL_API_VERSION=1.34
+export NOVA_VERSION=1.1
+export OS_PROJECT_NAME=Dev
+export OS_PASSWORD=default
+export OS_NO_CACHE=True
+export COMPUTE_API_VERSION=1.1
+export no_proxy=,10.250.23.52,10.250.11.39
```

Comment:
It can only work in lab env, but cannot work in other envs?
How about generating it dynamically with existing environment variables?